components/Layout.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import Head from 'next/head';
 import PropTypes from 'prop-types';
+import { Navbar, NavbarBrand } from 'reactstrap';
 
 export default class Layout extends React.Component {
   render() {
     return (
-      // create div for page content with height of the entire veiw port
       <div>
         <Head>
           <title> {this.props.title} </title>
+          <link
+            rel="stylesheet"
+            href="https://www.boston.gov/crispus/css/public.css"
```

Comment:
Should pull from `patterns.boston.gov`: `<link rel="stylesheet" type="text/css" href="https://patterns.boston.gov/css/public.css" />`

Reply:
👍